src/OrderFormAcceptanceTests.TestData/SupplierInfo.cs

```diff
@@ -94,6 +94,14 @@ public static int SupplierWithMoreThanOneSolution(string connectionString)
                 $@"SELECT SupplierId, COUNT(*) FROM CatalogueItem WHERE CatalogueItemTypeId = {(int)CatalogueItemType.Solution} GROUP BY SupplierId ORDER BY 2 DESC";
 
             return SqlExecutor.Execute<int>(connectionString, query, null).FirstOrDefault();
-        } 
+        }
+
+        public static string GetSupplierSolutionNameWithPrice(string connectionString, string supplierId)
+        {
+            var query =
+                $@"SELECT ci.[Name] FROM [buyingcatalogue].[dbo].[CatalogueItem] ci INNER JOIN CataloguePrice cp ON cp.CatalogueItemId=ci.CatalogueItemId WHERE ci.CatalogueItemTypeId = 1 AND ci.PublishedStatusId = 3 AND ci.SupplierId = @supplierId";
```

Comment:
The database name can be omitted as can the square brackets from the `FROM` clause. I'd also recommend including `AS` when defining your table aliases (I think it's clearer). I'd also recommend spaces around all operators and a terminating semicolon, so query would become:

```sql
 ci.[Name] FROM dbo.CatalogueItem AS ci INNER JOIN CataloguePrice AS cp ON cp.CatalogueItemId = ci.CatalogueItemId WHERE ci.CatalogueItemTypeId = 1 AND ci.PublishedStatusId = 3 AND ci.SupplierId = @supplierId;
```